app/models/spotlight/search.rb

```diff
@@ -10,24 +10,22 @@ class Spotlight::Search < ActiveRecord::Base
   scope :published, -> { where(on_landing_page: true) }
   validates :title, presence: true
   has_paper_trail
+  
+  belongs_to :masthead, dependent: :destroy
+  belongs_to :thumbnail, class_name: "Spotlight::FeaturedImage", dependent: :destroy
+  accepts_nested_attributes_for :thumbnail, update_only: true
+  accepts_nested_attributes_for :masthead, update_only: true
+
 
   before_create do
-    self.featured_item_id ||= default_featured_item_id
+    set_default_featured_image
   end
 
   include Blacklight::SolrHelper
   include Spotlight::Catalog::AccessControlsEnforcement
 
-  def featured_item
-    if self.featured_item_id.present?
-      @featured_item ||= get_solr_response_for_doc_id(self.featured_item_id).last
```

Comment:
Scratch that. In the UI, we're limiting the autocomplete to members of the browse category. I guess that's probably good enough.